/usr/local/lib/python3.10/dist-packages/gymnasium/envs/registration.py:519: DeprecationWarning: [33mWARN: The environment Hopper-v4 is out of date. You should consider upgrading to version `v5`.[0m
  logger.deprecation(
Traceback (most recent call last):
  File "/usr/local/lib/python3.10/dist-packages/gymnasium/envs/mujoco/mujoco_env.py", line 13, in <module>
    import mujoco
ModuleNotFoundError: No module named 'mujoco'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/root/DILLAB/SAC/run.py", line 82, in <module>
    main()
  File "/root/DILLAB/SAC/run.py", line 78, in main
    running_training_loop()
  File "/root/DILLAB/SAC/run.py", line 15, in running_training_loop
    env = make_env()
  File "/root/DILLAB/SAC/config.py", line 25, in make_env
    gym.make(
  File "/usr/local/lib/python3.10/dist-packages/gymnasium/envs/registration.py", line 704, in make
    env_creator = load_env_creator(env_spec.entry_point)
  File "/usr/local/lib/python3.10/dist-packages/gymnasium/envs/registration.py", line 551, in load_env_creator
    mod = importlib.import_module(mod_name)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/usr/local/lib/python3.10/dist-packages/gymnasium/envs/mujoco/__init__.py", line 1, in <module>
    from gymnasium.envs.mujoco.mujoco_env import MujocoEnv
  File "/usr/local/lib/python3.10/dist-packages/gymnasium/envs/mujoco/mujoco_env.py", line 15, in <module>
    raise error.DependencyNotInstalled(
gymnasium.error.DependencyNotInstalled: MuJoCo is not installed, run `pip install "gymnasium[mujoco]"`
